web/src/main/resources/public/src/app/registration/registration_user/registration.component.ts

```diff
@@ -119,14 +122,24 @@ export class RegistrationComponent implements OnInit {
     }
 
     onSubmitJoin() {
+    	if(this.captchaUser == null || this.captchaUser == '' ) {
+            this._toasterService.pop('error',' Ви не заповнили капчу');
+            return;
+        }
+        if(this.captcha != this.captchaUser) {
+            this._toasterService.pop('error',' Ви погано заповнили капчу');
+            return;
+        }
         this.SenderJoin();
     }
 
     SenderJoin(): any {
         let isSuccessful: boolean = false;
         this.registerService.registerUser(this.newUser)
             .subscribe(
-                data => {                
+                data => {
+                	this.sendEmailToUser();
```

Comment:
Can you refactor sentEmailToUser and sendEmailToGolova to single method sendTo(<User || String> to)?